fMRIVolume/scripts/multiEchoCombine.m

```diff
@@ -94,11 +95,28 @@ function multiEchoCombine(tscPath,TE,refPath,sctPath,T2starMethod,weightMethod,f
   nonlinAlgo = 'Levenberg-Marquardt'; 
 end
 
-% load data
-I = niftiread(tscPath);
-hdr = niftiinfo(tscPath);
+%% set nifti IO functions
+% use imaging processing toolbox utilities, or if those are not available, use FSL utilities 
+function out = out2(fun);[~,out] = fun();end
+function out = out3(fun);[~,~,out] = fun();end
+if isempty(which('niftiread'))
+  infoNIFTI = @(fName) struct('ImageSize',out2(@() read_avw(fName))','PixelDimensions',out3(@() read_avw(fName))');
+  readNIFTI = @(fName) read_avw(fName);
+  writeNIFTI = @(img,fName,hdr) save_avw(img,fName,'f',hdr.PixelDimensions);
+  if isempty(which('read_avw'))
```

Comment:
Again, it is more obvious to test that something exists before the lines that reference it.

Reply:
Fixed.